### PR TITLE
Specific view for open and nearby results

### PR DESCRIPTION
**Pull request by st3v3nhunt**
To do:

- [ ] Merge updated styles

### CODE REVIEW

File: app/middleware/setLocals.js [hunk 1]
```diff
@@ -2,6 +2,12 @@ const completeOriginalUrl = require('../lib/completeOriginalUrl');
 const backLinkUtils = require('../lib/backLinkUtils');
 
 function fromRequest(req, res, next) {
+  let displayOpenResults = false;
+  if (req.query.open) {
+    displayOpenResults = req.query.open.toLowerCase() === 'true';
+  }
+
```
Comment (markysoft) [inline app/middleware/setLocals.js]: this could be replaced with a ternary operator

File: docker-compose.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ services:
         NODE_ENV: development
     command: yarn run start-watch
     environment:
-      API_BASE_URL: 'http://nearby-services-api:3001'
+      API_BASE_URL: '${API_BASE_URL-http://nearby-services-api:3001}'
```
Comment (markysoft) [inline docker-compose.yml]: is the env var `API_BASE_URL` used in local development?

Comment (st3v3nhunt) [reply docker-compose.yml]: I have been using it so I can point at the latest version of the API.

**Comment by st3v3nhunt** [comment]
Closing this PR as it has been superseded by #432.
The feedback has been addressed in #433

**Pull request closed without merging**